src/applications/static-pages/static-pages-entry.js

```diff
@@ -131,3 +132,9 @@ if (location.pathname === '/disability-benefits/increase-claims-testing/') {
 if (location.pathname === '/') {
   createMyVALoginWidget(store);
 }
+
+/* eslint-disable no-unused-vars,camelcase */
+const lazyLoad = new LazyLoad({
```

Comment:
I'm on the fence about if this is really worth it. Curious about other thoughts.

Reply:
We could profile it against a simulated slow connection using the timeline tool (with screenshots enabled). This branch vs master locally.

Reply:
The results of testing this were that the JS isn't getting pushed back by the images, but lazily loading them appears to save about half a second (on simulated fast 3G) in when the page fully loads. Given that we also save users 150kb by not loading them if they just go to the homepage and open a menu, seems worth it.